src/Gui/WidgetFactory.cpp

```diff
@@ -466,24 +450,14 @@ Py::Object PythonWrapper::fromQObject(QObject* object, const char* className)
         return Py::asObject(pyobj);
     }
     throw Py::RuntimeError("Failed to wrap object");
-
-#elif QT_VERSION >= 0x050000
+#else
     // Access shiboken2/PySide2 via Python
     //
     return qt_wrapInstance<QObject*>(object, className, "shiboken2", "PySide2.QtCore", "wrapInstance");
-#else
-    // Access shiboken/PySide via Python
-    //
-    return qt_wrapInstance<QObject*>(object, className, "shiboken", "PySide.QtCore", "wrapInstance");
 #endif
-
 #if 0 // Unwrapping using sip/PyQt
     Q_UNUSED(className);
-#if QT_VERSION >= 0x050000
     return qt_wrapInstance<QObject*>(object, "QObject", "sip", "PyQt5.QtCore", "wrapinstance");
-#else
-    return qt_wrapInstance<QObject*>(object, "QObject", "sip", "PyQt4.Qt", "wrapinstance");
-#endif
 #endif

```

Comment:
I think you can remove this `#if`-`#endif` block

Reply:
There are three instances of that code in this file -- I am operating on the assumption that it was commented out but left in place for some reason, so I'm hesitant to remove it.

Reply:
I undestand but afaik, freecad changed from sip+pyqt to shiboken+pyside back in around 0.15.
This makes me believe that it should be safe to remove. Maybe it's best to tag in someone who knows this stuff better?

Reply:
I'm sure @wwmayer can tell us 😄

Reply:
(I've added a separate commit, 056bbfb, that does this removal, in the expectation that it turns out that it's fine to remove.)

Reply:
It's true that we moved to PySide a few years ago but the above code should stay as a fallback solution because PySide packages can be broken or do not even exist.